src/lightning_app/utilities/packaging/cloud_compute.py

```diff
@@ -66,13 +66,16 @@ class CloudCompute:
             For example 1100 will become 1024. If set to zero (the default) will get the default 64MiB inside docker.
 
         mounts: External data sources which should be mounted into a work as a filesystem at runtime.
+
+        preemptible: Whether to run on a preemptible machine.
```

Comment:
Are we set on calling it preemptible? Maybe just `spot` or `use_spot` is better?

Reply:
This is called preemptible in the platform and @williamFalcon asked in the past to revert spot to preemptible.